client-lib/defaults.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  id: '',
+  src: '',
+  url: '', // only used for <audio>, <video> tags, and soundcloud
+  domain: '',
+  minimized: false,
+  loaded: false,
+  error: false,
+  muted: false,
+  exited: false,
+  time: '0:00 / 0:00',
+  currentTime: 0,
+  duration: 0,
+  progress: 0.001, // force progress element to start out empty
+  playing: false,
+  volume: '0.5',
```

Comment:
I noticed in the debugger that React seems to convert this into a float. should it be set as a float here?